src/TypeResolver.php

```diff
@@ -130,26 +124,25 @@ public function resolve($type, Context $context = null)
     /**
      * Analyse each tokens and creates types
      *
-     * @param \ArrayIterator $tokens  the iterator on tokens
-     * @param Context        $context
-     * @param integer        $parserContext on of self::PARSER_* constants, indicating
```

Comment:
This description made sense.

Reply:
It looks like ECS chose to remove the `@param Context $context` tag, based on:
* no description in the tag
* typehint already in the method signature

It doesn't look like it altered any existing descriptions, other than leading spaces on the second line of `$parserContext`'s description.